src/braket/ahs/atom_arrangement.py

```diff
@@ -63,56 +69,153 @@ def __init__(self):
 
     def add(
         self,
-        coordinate: Union[tuple[Number, Number], np.ndarray],
+        coordinate: Union[Tuple[Number, Number], np.ndarray],
         site_type: SiteType = SiteType.FILLED,
-    ) -> AtomArrangement:
+    ) -> "AtomArrangement":
         """Add a coordinate to the atom arrangement.
-
```

Comment:
We are using Google formatted docstrings in this repo (see more here: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings), please don't remove these empty lines.

Reply:
Same for many other removed lines in the docstrings below.